flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -919,4 +922,76 @@ void getObject(DefaultMutableTreeNode parent, String objectRef) {
       }
     });
   }
+
+  /**
+   * JBTable custom selection and hover renderer for Classes table.
+   */
+  public class ClassesJBTable extends JBTable {
+    final private Color HOVER_BACKGROUND_COLOR = new Color(0xCFE6EF);
```

Comment:
use JBColor instead of Color everwhere so that colors look reasonable in both light and dark intellij themes.